renderer/routes/recent.svelte

```diff
@@ -0,0 +1,110 @@
+<script>
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+  import { _ } from 'svelte-intl'
+  import { replace } from 'svelte-spa-router'
+  import { distinct, filter, map } from 'rxjs/operators'
+  import { Heading, Button, PlaylistTracksTable } from '../components'
+  import { load, changes, removals } from '../stores/playlists'
+  import { add } from '../stores/track-queue'
+  import {
+    invoke,
+    formatTimeLong,
+    fromServerChannel,
+    sumDurations
+  } from '../utils'
+
+  let playlist
+
+  $: playlistId = 151120
+  $: if (!playlist || playlist.id !== playlistId) {
+    loadPlaylist()
+  }
+  $: total = playlist && playlist.trackIds ? playlist.trackIds.length : 0
+
+  async function loadPlaylist() {
+    playlist = await load(playlistId)
+    if (!playlist) {
+      return replace('/recent')
+    }
+  }
+
+  onMount(() => {
+    const changeSub = changes
+      .pipe(
+        map(changed => changed.find(({ id }) => id === playlistId)),
+        filter(n => n),
+        distinct()
+      )
+      .subscribe(async changed => {
+        if (!changed.tracks) {
+          // then load tracks if not yet available
+          playlist = await load(playlist.id)
+        } else {
+          // update now in case of media change
+          playlist = changed
+        }
+      })
+
+    const trackSub = fromServerChannel(`track-changes`).subscribe(changed => {
+      if (
+        playlist &&
+        changed.some(({ id }) => playlist.trackIds.includes(id))
+      ) {
+        loadPlaylist()
+      }
+    })
```

Comment:
This one is still needed, because it'll allow updating UI when removing a tracks that was listed as recent.

To test, you can move a track file out of the tracked folder on your hard drive.